bin/qdrant-docker-image-ecr-deployment-cdk.ts

```diff
@@ -1,21 +1,35 @@
 #!/usr/bin/env node
 import 'source-map-support/register';
+
 import * as cdk from 'aws-cdk-lib';
+import * as dotenv from 'dotenv';
 import { QdrantDockerImageEcrDeploymentCdkStack } from '../lib/qdrant-docker-image-ecr-deployment-cdk-stack';
 
+dotenv.config(); // Load environment variables from .env file
 const app = new cdk.App();
-new QdrantDockerImageEcrDeploymentCdkStack(app, 'QdrantDockerImageEcrDeploymentCdkStack', {
-  /* If you don't specify 'env', this stack will be environment-agnostic.
-   * Account/Region-dependent features and context lookups will not work,
-   * but a single synthesized template can be deployed anywhere. */
 
-  /* Uncomment the next line to specialize this stack for the AWS Account
-   * and Region that are implied by the current CLI configuration. */
-  // env: { account: process.env.CDK_DEFAULT_ACCOUNT, region: process.env.CDK_DEFAULT_REGION },
+const { CDK_DEFAULT_ACCOUNT: account, CDK_DEFAULT_REGION: region } = process.env;
+
+const cdkRegions = process.env.CDK_DEPLOY_REGIONS?.split(',') ?? [region]; // Parsing comma separated list of regions
+const environments = process.env.ENVIRONMENTS?.split(',') ?? ['dev']; // Parsing comma separated list of environments
+
+const DEFAULT_IMAGE_VERSION = 'latest';
 
-  /* Uncomment the next line if you know exactly what Account and Region you
-   * want to deploy the stack to. */
-  // env: { account: '123456789012', region: 'us-east-1' },
+for (const cdkRegion of cdkRegions) {
+    for (const environment of environments) {
+        new QdrantDockerImageEcrDeploymentCdkStack(app, `QdrantDockerImageEcrDeploymentCdkStack-${cdkRegion}-${environment}`, {
+            env: {
+                account,
+                region: cdkRegion,
+            },
+            tags: {
+                environment,
+            },
+            repositoryName: `${process.env.ECR_REPOSITORY_NAME}-${environment}` ?? 'qdrant-docker-image-ecr-deployment-cdk',
```

Comment:
**Suggestion:** Ensure that the repository name fallback is used when `process.env.ECR_REPOSITORY_NAME` is not set, not when the constructed name is falsy.
```suggestion
            repositoryName: process.env.ECR_REPOSITORY_NAME ? `${process.env.ECR_REPOSITORY_NAME}-${environment}` : 'qdrant-docker-image-ecr-deployment-cdk',
```